library/eclipse-mosquitto

```diff
@@ -1,7 +1,7 @@
-Maintainers: David Audet <david.audet@ca.com> (@daudetCA)
+Maintainers: Roger Light <roger@atchoo.org> (@ralight)
 GitRepo: https://github.com/eclipse/mosquitto.git
+GitCommit: 26e57661826f7dd3d965a5f8fea928d785a68b5b
+Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
```

Comment:
Doh, almost missed this one, the base image `alpine:3.8` doesn't support `arm32v7`.

Reply:
Well spotted! I've removed it and pushed the change.